### DISCUSSION TITLE
Sextupole RDT terms in pyAT

### DISCUSSION THREAD

**Discussion by xuanhaoz** [atcollab/at#777] [Python AT]
Dear All,
Is there an equivalent function in pyAT to <computeRDT> in MATLAB AT?
If not, are there plans to add this in the near future?

I'm hoping to include the RDTs in an optimisation algorithm written in python. For various reasons I can't just invoke the matlab engine in python. Any suggestions for a work around?

**Comment by swhite2401** [comment]
Hello,
You are correct for the moment we have not ported this functionality to python. This is on the todo list...
Have you tried compiling you matlab code into a shared library? These can be called in python directly in principle.
